libtransact/src/state/merkle/mod.rs

```diff
@@ -17,6 +17,48 @@
  * -----------------------------------------------------------------------------
  */
 
+//! Merkle-Radix Tree state.
+//!
+//! ## Merkle Hashes
+//!
+//! Transact provides an addressable Merkle-Radix tree to store data for state. Let’s break that
+//! down: The tree is a Merkle tree because it is a copy-on-write data structure which stores
+//! successive node hashes from leaf-to-root upon any changes to the tree. For a given set of state
+//! transitions, we can generate a single root hash which points to that version of the tree.
+//!
+//! The root hash can be used to verify that the same set of state transitions applied against the
+//! same state ID in another database will produce the same results.
+//!
+//! For more information about general concepts, see the [Merkle]() page on Wikipedia.
```

Comment:
Maybe strike these wikipedia references given this is code doc

Reply:
Removed